src/main/java/tests/OperationStrategyTest.java

```diff
@@ -0,0 +1,25 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
It's recommended to avoid wildcard imports. Import only the classes that are needed.